lib/assets/stylesheets/_govuk_tech_docs.scss

```
@@ -1,4 +1,15 @@
-$govuk-assets-path: "/assets/govuk/assets/" !default;
+@function tech-docs-image-url-function($filename) {
+  @return image-url($govuk-images-path + $filename)
+}
+
+@function tech-docs-font-url-function($filename) {
+  @return font-url($govuk-fonts-path + $filename)
+}
```

Minor but it looks like these need semicolons for the syntax highlighting to work correctly

```suggestion
@function tech-docs-image-url-function($filename) {
  @return image-url($govuk-images-path + $filename);
}

@function tech-docs-font-url-function($filename) {
  @return font-url($govuk-fonts-path + $filename);
}
```

---

Weird how the syntax highlighting works correctly in the diff 🤷🏻

---

Huh, weird. Thanks for catching that, I didn't realise Sass is semicolon-y.